cmd/clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -106,6 +116,23 @@ kustomize build $VSPHERE_CLUSTER_API_CRD_PATH/default/ > $PROVIDERCOMPONENT_GENE
 echo "---" >> $PROVIDERCOMPONENT_GENERATED_FILE
 kustomize build $CLUSTER_API_CRD_PATH/default/ >> $PROVIDERCOMPONENT_GENERATED_FILE
 
+if [ $USE_KUBECON = 1 ]; then
+  echo "Updating generated provider-components.yaml file with containers from kubecon 2018..."
+#  CONTROLLER_CONTAINER=$(grep cluster-api-controller ${PROVIDERCOMPONENT_GENERATED_FILE})
+  CONTROLLER_CONTAINER=`grep "image:.*cluster-api-controller" ${PROVIDERCOMPONENT_GENERATED_FILE} | awk '{ print $2 }'`
+  VSPHERE_PROVIDER_CONTAINER=`grep "image:.*vsphere-cluster-api" ${PROVIDERCOMPONENT_GENERATED_FILE} | awk '{ print $2 }'`
+
+  cat $PROVIDERCOMPONENT_GENERATED_FILE | \
```

Comment:
sed supports multiple `-e` args.  You can also use `-i` to edit the file in place:

```sh
sed -i "$PROVIDERCOMPONENT_GENERATED_FILE" \
    -e "s|$VSPHERE_PROVIDER_CONTAINER|$KUBECON_VSPHERE_PROVIDER|" \
    -e "s|$CONTROLLER_CONTAINER|$KUBECON_CONTROLLER|"
```

Reply:
-i doesn't work on Mac unless you install gnu sed.